Assignment6/CParkingLot.cpp

```diff
@@ -0,0 +1,95 @@
+#include "CParkingLot.h"
+
+
+CParkingLot::CParkingLot() {
+    std::cout << "How many places are in the parking lot?" << std::endl;
```

Comment:
Hmm.. I thought about writing some user interface for our group's solution. What do you think? If we do some GUI it'll be amazing :)